bin/rust-petstore.sh

```diff
@@ -27,6 +27,21 @@ fi
 
 # if you've executed sbt assembly previously it will use that instead.
 export JAVA_OPTS="${JAVA_OPTS} -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="generate -t modules/openapi-generator/src/main/resources/rust -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g rust -o samples/client/petstore/rust --additional-properties packageName=petstore_client --library=hyper $@"
 
-java ${JAVA_OPTS} -jar ${executable} ${ags}
+for spec_path in \
+  modules/openapi-generator/src/test/resources/2_0/petstore.yaml \
+  modules/openapi-generator/src/test/resources/3_0/rust/rust-test.yaml \
+  modules/openapi-generator/src/test/resources/2_0/fileResponseTest.json\
+  ; do
+  spec=$(basename "$spec_path" | sed 's/.yaml//' | sed 's/.json//' )
+
+  for library in hyper reqwest; do
+    ags="generate --template-dir modules/openapi-generator/src/main/resources/rust
```

Comment:
You probably want `args` not `ags`, and you probably want to check the return code.

Reply:
I'm not sure - `rust-server` has `args`, but everything else appears to use `ags`. I don't think it matters much, though, so will switch.

Good catch - we can no longer rely on implicitly returning a non-zero code on failure. I'll fix that.